duneanaobj/StandardRecord/StandardRecord.h

```diff
@@ -40,12 +50,30 @@ namespace caf
     int reco_q;
     float Elep_reco;
     float theta_reco;
+    int reco_lepton_pdg;
 
     float RecoLepEnNue;
     float RecoHadEnNue;
     float RecoLepEnNumu;
     float RecoHadEnNumu;
 
+    double pileup_energy;
+
+    SRNDLAr  ndlar;
+    SRTMS  ndtms;
```

Comment:
Thoughts on nesting one more level so we have `nd.lar` and `nd.tms`?

Reply:
not a bad idea.  probably there will be ND-level stuff we'll want in any case (maybe `nd.mpdpos` or some such to track where the MPD is off-axis-wise, for example) which would probably want to hang directly off `nd` anyway.

Reply:
implemented in 4eb84e9d